pheromone-driver.py

```diff
@@ -0,0 +1,33 @@
+from pheromone import pheromone
+import numpy as np
+import moves
+import random
+np.set_printoptions(formatter={'float': lambda x: "{0:0.3f}".format(x)})
+
+def getObstacleData(size,prob):
```

Comment:
Consider using the new map implementation with mock objects stores to store obstacles. It might be helpful later